community.md

```diff
@@ -72,6 +72,16 @@ Some quick tips when using email:
 and include only a few lines of the pertinent code / log within the email.
 - No jobs, sales, or solicitation is permitted on the Apache Spark mailing lists.
 
+<a name="chat"></a>
+<h4>Chat Rooms</h4>
+
+Chat rooms are great for quick questions or discussions on specialized topics. The following chat rooms are not officially part of Apache Spark; they are provided for reference only.
+<ul>
+  <li>
+    <a href="https://gitter.im/spark-scala/Lobby">Spark with Scala</a> is for questions and discussions related to using Spark with the Scala programming language.
```

Comment:
Yes, I think linking to 'unofficial' chats is OK if they're active and useful and any such resource is linked. 

I have no idea what the theory is about HTML vs markdown but this can use simple `- ...` syntax because the other docs do. I don't feel strongly. HTML is used inconsistently.

I will probably want to clean up the Community menu along with some other changes here because it's getting busy and "Mailing Lists" no longer described what is really linked here. I'll do that separately.